### ISSUE TITLE
Supernumerary tracheal bronchus

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6629]
**Preferred term label:** Supernumerary tracheal bronchus

**Synonyms**

**Definition (free text, please give PubMed ID)**
Accessory bronchus which exits the trachea in addition to an anatomically normal branching upper lobe bronchus. PMID: 11158647, PMID: 19332762

**Parent term (use hpo.jax.org/app)**
Tracheal bronchus (once created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
